jest.config.js

```diff
@@ -2,6 +2,8 @@ const func = require("@jupyterlab/testutils/lib/jest-config");
 module.exports = {
   ...func(__dirname),
   testRegex: ["/test/.*.spec.js[x]?$"],
-  transformIgnorePatterns: ["node_modules/(?!(@jupyterlab|@jupyter-widgets)/)"],
+  transformIgnorePatterns: [
+    "node_modules/(?!(@jupyterlab|@jupyter-widgets|y-protocols|lib0)/)",
+  ],
```

Comment:
@stevejpurves jest was failing on checking some packages in node_modules (not quite sure why it needs to touch dependencies), but this was our bandaid for the errors. Is the is the right thing to do?

Reply:
@moorepants yes it's the right thing to do, for now anyway. Basically, `jest` does not consume ES6 modules and needs some of the dependencies that `thebelab.js` pulls in transpiled. 

It is not great as this list of things in `transformIgnorePatterns` can grow over time, but it's manageable for now. It's a fundamental problem I think with jest and we'd need to step back and look at module loading stuff and may even more to typescript based builds to get around it.